packages/tables/resources/views/index.blade.php

```diff
@@ -750,15 +751,19 @@ class="filament-table-header-cell-{{ \Illuminate\Support\Str::of($column->getNam
                                 @endif
 
                                 @if ($isSelectionEnabled)
-                                    <x-tables::checkbox.cell :class="\Illuminate\Support\Arr::toCssClasses([
-                                        'hidden' => $isReordering,
-                                    ])">
-                                        <x-tables::checkbox
-                                            x-model="selectedRecords"
-                                            :value="$recordKey"
-                                            class="filament-tables-record-checkbox"
-                                        />
-                                    </x-tables::checkbox.cell>
+                                    @if ($recordBulkActionEnabled)
```

Comment:
Add here for later evaluation:

```suggestion
                                    @if ($isRecordBulkActionEnabled($record))
```